code/data_structures/src/list/singly_linked_list/operations/find/find.cpp

```diff
@@ -0,0 +1,78 @@
+#include <iostream>
+
+// Part of Cosmos by OpenGenus Foundation
+struct Node
+{
+    Node *nextNode;
+    int data;
+
+    Node() //constructor
+    {
+        nextNode = NULL;
+        data = 0;
+    }
+};
+
+// Singly Linked List
+struct LinkedList
```

Comment:
Why not use a class with some member functions?

Reply:
Just preference, doesn't matter to me.